virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.adoc

```diff
@@ -4,7 +4,7 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-viewing-qemu-guest-agent-web
 toc::[]
 
-The QEMU guest agent runs on the virtual machine and passes information to the host about the virtual machine, users, and file systems. You can view this information in the web console.
+When the QEMU guest agent runs on the virtual machine, you can use the web console to view information about the virtual machine, users, and file systems.
```

Comment:
and secondary networks

Reply:
Updated the content by adding info on secondary networks.